angular.js

```diff
@@ -4,5 +4,59 @@ module.exports = {
         'plugin:@angular-eslint/recommended--extra',
         'plugin:@angular-eslint/template/process-inline-templates',
         './typescript'
-    ]
+    ],
+    rules: {
+        /*
+            Overrides to Angular recommended rules:
+            https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.json
+        */
+
+        '@angular-eslint/component-class-suffix': 'error',
+        '@angular-eslint/contextual-lifecycle': 'error',
+        '@angular-eslint/directive-class-suffix': 'error',
+        '@angular-eslint/no-conflicting-lifecycle': 'error',
+        '@angular-eslint/no-empty-lifecycle-method': 'error',
+        '@angular-eslint/no-host-metadata-property': 'error',
+        '@angular-eslint/no-input-rename': 'error',
```

Comment:
We discussed this in person. General consensus is that we can probably disable this rule because changing input names is a deliberate act which is almost impossible to do accidentally. @TrevorKarjanis will try out this rule in a codebase and create a proposal for voting.

Reply:
In testing, exceptions are common for directives that need to prefix their inputs to avoid conflicting with the inputs of other directives. Rather than disable the rule, I propose to allow this rule be disabled for the files of these directives.

**Proposal**
```javascript
/*
    Disallow renaming directive inputs generally. Use an inline comment to disable the rule for exceptions where the 
    context changes significantly, or disable the rule for files of directives that need to prefix their inputs to avoid conflicts.
*/
'@angular-eslint/no-input-rename': 'error'
```

Reply:
@TrevorKarjanis can you give more detail about why your proposal is to keep the rule set to `'error'`? The cases you advocate for disabling it seem valid and our sense last week was that devs were unlikely to rename by accident. What led you to say that the rule is worth enabling?

Reply:
It was found that renaming inputs are not common, and therefore I don't currently have sufficient argument to disable the rule. 

We have decided to accept this proposal.